### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Timi2424**
- [DEMO LINK](https://<Timi2424>.github.io/js_promise_basic_DOM/)

I need explanation why then method for promise2 should be empty, or is it becouse the way the tests were written

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise 1 resolved!');
```
Comment (Barosz30) [inline src/scripts/main.js:5]: You dont realy need it here. It's realy added to site bit later.

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise 2 rejected!'));
```
Comment (Barosz30) [inline src/scripts/main.js:11]: Same as above. There's no reason to use that here, if you put that to "message error-message" later.

**Review by Barosz30 — APPROVED** [review]
Overall, it looks ok. Tough i cant Promise. 

Pun intended

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,3 +1,40 @@
...
+  .catch(() => {
+    const errorMessageDiv = document.createElement('div');
+
+    errorMessageDiv.className = 'message error-message';
+    errorMessageDiv.textContent = 'Promise was rejected!';
+    document.body.appendChild(errorMessageDiv);
+  });
+
+promise2
+  .then(() => {
```
Comment (mbruhler) [inline src/scripts/main.js:32]: You don't need this here if it will be rejected, just a catch method

File: src/scripts/main.js (hunk 2 above)
Comment (mbruhler) [inline src/scripts/main.js:11]: you should just `reject()` or pass this argument later when you call this promise

File: src/scripts/main.js (hunk 1 above)
Comment (mbruhler) [inline src/scripts/main.js:5]: You should just `resolve` or pass this argument later when you call this promise

**Review by mbruhler — CHANGES_REQUESTED** [review]
There are couple of fixes to go.

Also work on styling a little bit, logo should be logo, and error / success should be also visibly changed

**Pull request closed without merging**